linodecli/plugins/obj/__init__.py

```diff
@@ -527,7 +528,15 @@ def call(
 
         sys.exit(2)  # requirements not met - we can't go on
 
-    parser = get_obj_args_parser()
+    clusters = [
+        c["id"]
+        for c in _do_get_request(  # pylint: disable=protected-access
+            context.client.config.base_url,
+            "/object-storage/clusters",
+            token=context.client.config.get_value("token"),
```

Comment:
This doesn't seem to be working with `LINODE_CLI_TOKEN` environment variable.

```
linode-cli % lin obj --cluster us-west --help
Could not contact https://api.linode.com/v4beta/object-storage/clusters - Error: 401
```

Reply:
This seems to be a separated issue that's also impacting `linode obj configure`. But usage of `--cluster` is much more common than `configure`.

Reply:
I put up a fix here: https://github.com/jriddle-linode/linode-cli/pull/2/
Do you think that would work?

Reply:
Good catch, merged changes, thank you.